pinot/generative/torch_gvae/layers.py

```diff
@@ -0,0 +1,61 @@
+import torch
+import torch.nn.functional as F
+from torch.nn.modules.module import Module
+from torch.nn.parameter import Parameter
+import dgl
+import dgl.function as fn
+from dgl import DGLGraph
+
+gcn_msg = fn.copy_src(src='h', out='m')
+gcn_reduce = fn.sum(msg='m', out='h')
+
+"""
+Standalone convolution layers, separate from representation.dgl_legacy
```

Comment:
Could you comment on how is this different from the vanilla Kipf and Welling Graph Conv? is it the explicit dependency on adjacent matrix that's different?

Reply:
This has been removed from updated code